### PR TITLE
Get origin statically

### LINKED ISSUE #744: Get origin from environment
It's not necessary to use dynamic code to determine the origin as we are doing it, because we can probably get it from the environment.

### PR DESCRIPTION
**Pull request by Entkenntnis**
Fixes #744

### CODE REVIEW

**Comment by Entkenntnis** [comment]
Hm, something is not working out yet, https://frontend-git-use-static-origin.serlo.now.sh/origintest is showing the wrong origin. :face_with_head_bandage:

**Comment by elbotho** [comment]
@Entkenntnis did you see the failed "Env / production" test?

**Comment by Entkenntnis** [comment]
Thanks, this is now working!

**Comment by inyono** [comment]
Cool. Failing CI is because you changed `.env.production`. For that, you need to update the checksum in `.github/workflows/env.yaml` accordingly with `sha256sum .env.production`.

**Comment by Entkenntnis** [comment]
@elbotho You said that you are interested in the refactorings. Can you make a review of this PR? Thanks!

File: README.md [hunk 1]
```diff
@@ -914,4 +914,4 @@ The brother can pass a message to its sister by declaring the state in the paren
 ### How can I change the port of the dev server?
 
 You can change the port by running `yarn dev --port 8080`.
-Beware that authentication only works on the default port.
+Beware that some important features rely on the default port (authentication, backend fetch). Change port on your own risk.
```
Comment (elbotho) [inline README.md:917]: I love how you found something useful to trigger the deploy 😄

File: src/helper/frontent-origin.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const frontendOrigin =
+  process.env.NEXT_PUBLIC_ENV === 'production'
+    ? // we are on production branch
```
Comment (elbotho) [inline src/helper/frontent-origin.ts]: production includes production and staging, right?
maybe the comment could say that.

**Review by elbotho — APPROVED** [review]
Thanks! Great start for the refactors.

File: package.json [hunk 3]
```diff
@@ -43,7 +43,6 @@
     "jwt-decode": "^3.0.0",
     "katex": "^0.12.0",
     "next": "^10.0.0",
-    "next-absolute-url": "^1.0.0",
```
Comment (elbotho) [inline package.json:46]: ah maybe you could run `yarn` again to update yarn.lock

File: src/helper/frontent-origin.ts (hunk 2 above)
Comment (elbotho) [reply src/helper/frontent-origin.ts]: (I'd do it, but i'm not sure I'm right :)

Comment (Entkenntnis) [reply src/helper/frontent-origin.ts]: Actually, in this case production is only production (e.g. de.serlo.org), maybe you have an idea how to make this clearer?

Comment (Entkenntnis) [reply src/helper/frontent-origin.ts]: production branch -> production environment?

Comment (elbotho) [reply src/helper/frontent-origin.ts]: Is staging handled in the second case then?

Comment (elbotho) [reply src/helper/frontent-origin.ts]: (in other words I don't understand how staging is handled here atm. :)

Comment (Entkenntnis) [reply src/helper/frontent-origin.ts]: staging is technically a preview deployment (we have no custom domain here)

Comment (elbotho) [reply src/helper/frontent-origin.ts]: Thanks for clarifying!

**Comment by elbotho** [comment]
👍

**Pull request merged**